doc/plugins/Working with plugins.md

```diff
@@ -1,7 +1,18 @@
 # Working with plugins
 
-# Core plugins
+Plugins usually add global-level functionality to Vue. In Vue Storefront there are two types of plugins.
+
+## Core plugins
+
+Core plugins are located in `src/components` folder. There are two core plugins:
```

Comment:
Small mistake. You mean they are in `src/plugins`?

Reply:
Fixed